R/navs-legacy.R

```diff
@@ -161,13 +161,15 @@ navset_bar <- function(
     fluid = fluid,
     position = .navbar_options$position,
     bg = .navbar_options$bg,
-    inverse = .navbar_options$inverse,
+    inverse = .navbar_options$type,
     collapsible = .navbar_options$collapsible,
     underline = .navbar_options$underline,
     # theme is only used to determine whether legacy style markup should be used
     # (and, at least at the moment, we don't need legacy markup for this exported function)
     theme = bs_theme()
   )
+
+  navbar_options_apply_attribs(navbar, .navbar_options)
```

Comment:
I think `navbar_options_apply_attribs()` could be easier to reason about if you assume that it gets passed a tag object and a list of attributes.

```suggestion
  navbar_options_apply_attribs(navbar[[1]], .navbar_options$attribs)
```

Reply:
Ok, yea, I see where you're coming from. Feel free to resolve this one

Reply:
I moved all of the logic of `navbar_options_apply_attribs()` into `navs_bar_()`, which cleans up a lot of this logic and removed the need for a few other comments that I've resolved.

Reply:
Nice, much better, thanks!